src/app/base/docsUrls.ts

```diff
@@ -29,6 +29,8 @@ const docsUrls = {
     "https://maas.io/docs/how-to-work-with-tags#heading--kernel-options",
   tagsXpathExpressions:
     "https://maas.io/docs/how-to-work-with-tags#heading--xpath-expressions",
+  vaultIntegration:
+    " https://maas.io/docs/how-to-integrate-with-vault#heading--enabling-vault",
```

Comment:
```suggestion
    "https://maas.io/docs/how-to-integrate-with-vault#heading--enabling-vault",
```

Reply:
Do you know when this will become available? Might be worth commenting out and adding a TODO comment with a link to a task to update once it's done (otherwise docs tests will keep failing).

Reply:
I'll have to ask Bill. I can comment out the link for now and just have it direct to the docs homepage?